static/js/settings_org.js

```diff
@@ -857,6 +830,10 @@ function check_property_changed(elem, for_realm_default_settings) {
                 "#org-notifications .language_selection_widget .language_selection_button span",
             ).attr("data-language-code");
             break;
+        case "emojiset":
+        case "user_list_style":
+            proposed_val = get_input_element_value($elem, "radio-group");
```

Comment:
This can probably be removed (and also the case block of message limit settings) if `get_input_element_value` uses the type from `data-setting-widget-type` attribute instead of passing `typeof current_val` in the default block. But will need to make sure it does not break other settings, so can do it separately if we want.

Reply:
Yeah, let's do this in a follow-up; I think it is a good idea though.